llvm/test/DebugInfo/AArch64/instr-ref-target-hooks-sp-clobber.mir

```diff
@@ -0,0 +1,245 @@
+# Test to ensure that variable "__last" is properly recovered at the end of the livedebugvalues pass when Instruction Referencing-based LiveDebugValues is used.
+# This testcase was obtained by looking at FileCheck.cpp and reducing it down via llvm-reduce.
+# This test is the same as llvm/test/DebugInfo/AArch64/instr-ref-target-hooks.ll, however, the output is taken just before the livedebugvalues pass, and then a clobber
+# to the stack slot has been added on line 238, the livedebugvalues pass should still recover the value, as it was loaded into $x8 before the clobber.
```

Comment:
IMO best to name the block rather than the line number, as the line number is going to drift over time. Putting a comment inside the MIR will help too.